cmd/do/do.go

```diff
@@ -321,6 +327,12 @@ func (verb *goVerb) Run(ctx context.Context, flags flag.FlagSet) error {
 	return nil
 }
 
+func (verb *jdocVerb) Run(ctx context.Context, flags flag.FlagSet) error {
```

Comment:
It might also be interesting to see if there's any free javadoc hosting sites like [godoc](https://godoc.org/github.com/google/gapid). I found [javadoc.io](http://www.javadoc.io/), but that requires the code to be in a Central Maven Repository.